Sources/Conformance/main.swift

```diff
@@ -60,6 +60,13 @@ func buildResponse(serializedData: Data) -> Conformance_ConformanceResponse {
         return response
     }
 
+    // Detect when something gets added to the conformance request that isn't
+    // supported yet.
+    if !request.unknownFields.data.isEmpty {
```

Comment:
Nit: `guard request.unknownFields.data.isEmpty else { ... }` is ever-so-slightly clearer since it's an exceptional early exit case.

Reply:
done